src/Uno.UI/UI/Xaml/FrameworkElement.Interface.wasm.cs

```diff
@@ -206,7 +224,12 @@ private void RecalculateBrushOnSizeChanged(bool shouldRecalculate)
 
 		public bool IsEnabled
 		{
-			get { return (bool)GetValue(IsEnabledProperty); }
+			get
+			{
+				Console.WriteLine($"{GetHashCode():X8} Get cached IsEnabled: {_isEnabledPropertyBackingField}");
```

Comment:
Codacy found an issue: [Remove this logging statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5714227)